src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1

```diff
@@ -100,8 +100,8 @@ CmdletsToExport = 'Add-AzureKeyVaultCertificate',
                'Undo-AzureKeyVaultKeyRemoval', 'Undo-AzureKeyVaultSecretRemoval', 
                'Add-AzureKeyVaultKey', 'Remove-AzureKeyVaultKey', 
                'Remove-AzureKeyVaultSecret', 'Restore-AzureKeyVaultKey', 
-               'Set-AzureKeyVaultKeyAttribute', 'Set-AzureKeyVaultSecret', 
-               'Set-AzureKeyVaultSecretAttribute', 
+               'Update-AzureKeyVaultKey', 'Set-AzureKeyVaultSecret',
+			   'Update-AzureKeyVaultSecret',
```

Comment:
@maddieclayton nit: I would run `Update-ModuleManifest` on this file to fix the formatting

Reply:
Done